examples/network/envoy.yaml

```diff
@@ -53,7 +53,7 @@ static_resources:
 
   clusters:
     - name: web_service
-      connect_timeout: 0.25s
+      connect_timeout: 5000s
```

Comment:
this is unrealistic and at least we should definitely add comment. Do we really need 5000s now?

Reply:
No, actually I checked its necessity and I found it isn't needed. Only the client needs to set timeout to 5000s over 4KiB. I'll revert it.

Reply:
The timeout error hasn't occurred when the client's timeout is set to 5000s.

**after removing timeout settings from both envoy and client**
```
cat loadtest20210812203958.log | head -n 180                                                                                                [20:43:59]
Creating proxy-wasm-go-sdk-devenv_sandbox_run ... 
Creating proxy-wasm-go-sdk-devenv_sandbox_run ... done
Creating proxy-wasm-go-sdk-devenv_sandbox_run ... 
Creating proxy-wasm-go-sdk-devenv_sandbox_run ... done
go test -v ./e2e -count=1
=== RUN   Test_dispatch_call_on_tick
--- PASS: Test_dispatch_call_on_tick (0.58s)
=== RUN   Test_foreign_call_on_tick
--- PASS: Test_foreign_call_on_tick (0.16s)
=== RUN   Test_helloworld
--- PASS: Test_helloworld (1.08s)
=== RUN   Test_http_auth_random
--- PASS: Test_http_auth_random (1.17s)
=== RUN   Test_http_body
=== RUN   Test_http_body/append
=== RUN   Test_http_body/prepend
=== RUN   Test_http_body/replace
=== RUN   Test_http_body/invalid
--- PASS: Test_http_body (2.11s)
    --- PASS: Test_http_body/append (0.50s)
    --- PASS: Test_http_body/prepend (0.50s)
    --- PASS: Test_http_body/replace (0.50s)
    --- PASS: Test_http_body/invalid (0.50s)
=== RUN   Test_http_headers
--- PASS: Test_http_headers (0.11s)
=== RUN   Test_http_routing
--- PASS: Test_http_routing (0.11s)
=== RUN   Test_metrics
--- PASS: Test_metrics (0.12s)
=== RUN   Test_network
--- PASS: Test_network (0.11s)
=== RUN   Test_shared_data
--- PASS: Test_shared_data (0.12s)
=== RUN   Test_shared_queue
--- PASS: Test_shared_queue (0.11s)
=== RUN   Test_vm_plugin_configuration
--- PASS: Test_vm_plugin_configuration (0.10s)
=== RUN   Test_http_load_with_high_bandwidth
11:40:37 I httprunner.go:81> Starting http test for http://localhost:18000 with 0 threads at 0.0 qps
...
11:41:01 I periodic.go:558> T002 ended after 24.0123772s : 25 calls. qps=1.0411297387082525
11:41:01 I periodic.go:558> T000 ended after 24.0125686s : 25 calls. qps=1.0411214400445272
11:41:01 I periodic.go:558> T001 ended after 24.012632s : 25 calls. qps=1.0411186911955341
11:41:01 I periodic.go:558> T003 ended after 24.0126469s : 25 calls. qps=1.04111804517477
11:41:01        numCalls = 1, payloadSize = 262144 [byte]
11:41:01 I httprunner.go:81> Starting http test for http://localhost:18000 with 0 threads at 0.0 qps
11:41:01 W periodic.go:321> Increasing the number of calls to the minimum of 2 with 1 thread. total duration will increase
11:41:01 I periodic.go:558> T000 ended after 5.5316ms : 1 calls. qps=180.7795212958276
11:41:01        Result: &{{HTTP  2021-08-12 11:41:01.573011 +0000 UTC m=+29.985006701 8 exactly 1 calls 179.06706061420002 5.5845ms 1 dev 0xc0003e00c0 1 false} <nil> map[200:1] 0xc0000121e0 0xc000012230 0xc0003e0180 0xc0003e0120 http://localhost:18000 1 -1 0xc00050e520}
11:41:01        numCalls = 1, payloadSize = 524288 [byte]
11:41:01 I httprunner.go:81> Starting http test for http://localhost:18000 with 0 threads at 0.0 qps
11:41:01 W periodic.go:321> Increasing the number of calls to the minimum of 2 with 1 thread. total duration will increase
11:41:01 I periodic.go:558> T000 ended after 12.0472ms : 1 calls. qps=83.00683976359653
11:41:01        Result: &{{HTTP  2021-08-12 11:41:01.5825025 +0000 UTC m=+29.994498201 8 exactly 1 calls 82.65965712774224 12.0978ms 1 dev 0xc000112420 1 false} <nil> map[200:1] 0xc000012050 0xc0000120a0 0xc0001124e0 0xc000112480 http://localhost:18000 1 -1 0xc00027c0b0}
11:41:01        numCalls = 1, payloadSize = 1048576 [byte]
11:41:01 I httprunner.go:81> Starting http test for http://localhost:18000 with 0 threads at 0.0 qps
11:41:01 W periodic.go:321> Increasing the number of calls to the minimum of 2 with 1 thread. total duration will increase
11:41:01 I periodic.go:558> T000 ended after 163.7644ms : 1 calls. qps=6.106333244587956
11:41:01        Result: &{{HTTP  2021-08-12 11:41:01.5996344 +0000 UTC m=+30.011630101 8 exactly 1 calls 6.104167620442857 163.8225ms 1 dev 0xc000112240 1 false} <nil> map[200:1] 0xc0000123c0 0xc0000124b0 0xc000112420 0xc0001123c0 http://localhost:18000 1 -1 0xc00027c400}
11:41:01        numCalls = 1, payloadSize = 2097152 [byte]
11:41:01 I httprunner.go:81> Starting http test for http://localhost:18000 with 0 threads at 0.0 qps
11:41:01 W periodic.go:321> Increasing the number of calls to the minimum of 2 with 1 thread. total duration will increase
11:41:02 I periodic.go:558> T000 ended after 1.0054664s : 1 calls. qps=0.9945633190726214
11:41:02        Result: &{{HTTP  2021-08-12 11:41:01.7718298 +0000 UTC m=+30.183825501 8 exactly 1 calls 0.9945059513225138 1.0055244s 1 dev 0xc0003e0060 1 false} <nil> map[200:1] 0xc00028c000 0xc00028c050 0xc0003e0180 0xc0003e0120 http://localhost:18000 1 -1 0xc00027c2e0}
11:41:02        numCalls = 1, payloadSize = 4194304 [byte]
11:41:02 I httprunner.go:81> Starting http test for http://localhost:18000 with 0 threads at 0.0 qps
11:41:02 W periodic.go:321> Increasing the number of calls to the minimum of 2 with 1 thread. total duration will increase
11:41:05 E http_client.go:775> Read error &{{0xc00025c280}} 127.0.0.1:18000 0 : read tcp 127.0.0.1:57722->127.0.0.1:18000: i/o timeout
11:41:05 I httprunner.go:63> Aborted run because of code -1 - data 
11:41:05 I periodic.go:558> T000 ended after 3.0028809s : 1 calls. qps=0.33301354043045794
11:41:05        Result: &{{HTTP  2021-08-12 11:41:02.7958259 +0000 UTC m=+31.207821601 8 exactly 1 calls 0.33301176607162236 3.0028969s 1 dev 0xc0003e0180 1 false} <nil> map[-1:1] 0xc000012050 0xc0000120a0 0xc0003e0240 0xc0003e01e0 http://localhost:18000 1 -1 0xc00050e070}
    load_test.go:70: 
                Error Trace:    load_test.go:70
                Error:          Not equal: 
                                expected: 1
                                actual  : 0
                Test:           Test_http_load_with_high_bandwidth
                Messages:       [2021-08-12 11:40:37.485][2990][info][main] [source/server/server.cc:330] initializing epoch 0 
...
```

Reply:
done